qiskit/ignis/measurement/discriminator/iq_discrimination.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+import numpy as np
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis
+from sklearn.discriminant_analysis import QuadraticDiscriminantAnalysis
+
+from qiskit.ignis.characterization.fitters import BaseFitter
+from qiskit.exceptions import QiskitError
+from qiskit.result.models import ExperimentResult
+from qiskit.result import postprocess
+
+
+class IQDiscriminationFitter(BaseFitter):
```

Comment:
Is `IQDiscriminationFitter` used rather than `DiscriminationFitter` because you could foresee something like a `RawDiscriminationFitter` that directly discriminates raw data?

Reply:
Yes. Not all of quantum is IQ. Think of ions for instance, the readout is quite different.